core/src/main/java/com/capitalone/dashboard/config/collector/CloudConfig.java

```diff
@@ -9,7 +9,7 @@
 
 public class CloudConfig extends CollectorItem {
     private static final String CLOUD_PROVIDER = "provider";
-    private static final List<NameValue> tags = new ArrayList<>();
+    private static final List<NameValue> TAGS = new ArrayList<>();
```

Comment:
because it is static final PMD says it should be all CAPS

<img width="1381" alt="screenshot 2016-07-01 08 02 50" src="https://cloud.githubusercontent.com/assets/14372687/16525713/8eb88946-3f62-11e6-86e6-f49e028eed09.png">